makeflow/src/dag_file.c

```diff
@@ -84,4 +87,32 @@ int dag_file_in_trans( const struct dag_file *f )
 		return 0;
 }
 
+int dag_file_mount_clean(struct dag *d) {
```

Comment:
After looking at this again I don't think it is necessary. I think it is appropriate to free this information, but only in the context of freeing a whole dag_file and not just mount information. Is there any practical point, other than code/memory cleanliness, to free this information?